src/internal.c

```diff
@@ -7990,16 +7990,12 @@ int ProcessPeerCerts(WOLFSSL* ssl, byte* input, word32* inOutIdx,
             while (listSz) {
                 word32 certSz;
 
+                if (args->totalCerts >= ssl->verifyDepth) {
```

Comment:
I changed this for nginx tests to pass.
If this doesn't work in your tests then use the WOLFSSL_NGINX define for the previous behaviour.

Reply:
Updated it. Thanks